tests/ut/test_decorators.py

```diff
@@ -434,6 +465,38 @@ def test_get_cache_keys_with_key_builder(self, decorator):
         with pytest.raises(Exception):
             assert await decorator_call(keys=[])
 
+    @pytest.mark.asyncio
+    async def test_disable_read(self, decorator, decorator_call):
+        decorator.cache.multi_get = CoroutineMock(return_value=[1, 2])
```

Comment:
same, not needed because its not being called?

Reply:
Removed